skbio/sequence/_sequence.py

```diff
@@ -373,6 +376,7 @@ def metadata(self):
         return self._metadata
 
     @metadata.setter
+    @stable(as_of="0.4.0")
```

Comment:
From grepping (`grep -r '\.setter' *`) there is one more: `DissimilarityMatrix.ids`.